src/mlpack/methods/ann/loss_functions/huber_loss.hpp

```diff
@@ -0,0 +1,110 @@
+/**
+ * @file huber_loss.hpp
+ * @author Mrityunjay Tripathi
+ *
+ * Definition of the Huber loss function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LOSS_FUNCTION_HUBER_LOSS_HPP
+#define MLPACK_METHODS_ANN_LOSS_FUNCTION_HUBER_LOSS_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The Huber loss is a loss function used in robust regression,
+ * that is less sensitive to outliers in data than the squared error loss.
+ * This function is quadratic for small values of \f$ y - f(x) \f$,
+ * and linear for large values, with equal values and slopes of the different
+ * sections at the two points where \f$ |y - f(x)| = delta \f$.
+ *
+ * @tparam ActivationFunction Activation function used for the embedding layer.
```

Comment:
Seems like this was included by mistake

Reply:
Ohh yes. It will be removed in the next commit.